airgun/views/hostcollection.py

```diff
@@ -57,9 +73,236 @@ def is_displayed(self):
     class details(SatTab):
         name = EditableEntry(name='Name')
         description = EditableEntry(name='Description')
+        content_host_limit = EditableLimitEntry(name='Content Host Limit')
+        # Package Installation, Removal, and Update
+        manage_packages = Text(".//li[@bst-feature-flag='remote_actions']"
+                               "/a[@ng-click='openPackagesModal()']")
```

Comment:
nitpick: are there other links on the page with ng-click='openPackagesModal()' and similar? imo locators like `.//a[@ng-click='openPackagesModal()']` would be enough for packages/errata/environment links